_data/engine-cli/docker_ps.yaml

```diff
@@ -107,6 +107,24 @@ examples: |-
   container that exposes TCP ports `100, 101, 102` displays `100-102/tcp` in
   the `PORTS` column.
 
+  ### Show disk usage by container
```

Comment:
oh! the docker cli reference documentation YAML files are generated from markdown files in the docker/cli repository; could you make the changes in that repository https://github.com/docker/cli/blob/master/docs/reference/commandline/ps.md ?

(let me know if you want me to take care of that 🤗 )

Reply:
Oh, oops. I glanced over the comments which explained this and I think I concluded that I was editing  in the right place because the file was not mentioned in dont_edit_here.yml or whatever.   I can make a new PR with the edit in the correct place.

Reply:
If these files are generated why are they checked in ?  

maybe I dont wanna get into the weeds here... its not my house lol

Reply:
At very least these files should include a header like 

```
#  ATTENTION: this is an auto-generated file. Please edit the original source file at: ...
```